keyboards/viktus/minne/keymaps/via/minne.json

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
via config files are not accepted to this repo

Reply:
Just the json file or all of them? I ask because I checked other boards and they had via keymaps.

Reply:
Sorry, just the .json file. It should be submitted to <https://github.com/the-via/keyboards> and not qmk_firmware.